packages/fx-core/resource/package.nls.json

```diff
@@ -266,6 +269,8 @@
   "plugins.apiConnector.reuseAppOption.placeholder": "Use the AAD app registration created for current Teams app",
   "plugins.apiConnector.anotherAppOption.title": "Use another app registration",
   "plugins.apiConnector.anotherAppOption.placeholder": "Provide information for your AAD app registration",
+  "plugins.apiConnector.headerLocationOption.title": "Request header",
```

Comment:
can we rename it to something like plugins.apiConnector.`requestHeaderOption`.title? the current name seems a little confusing since it's not location for header. Please also help see how to better describe `plugins.apiConnector.queryLocationOption.title`